bin/qds.py

```diff
@@ -183,107 +183,129 @@ def checkargs_cluster_id_label(args):
-                               arguments.disallow_cluster_termination,
-                               arguments.enable_ganglia_monitoring,
-                               arguments.node_bootstrap_file,)
-
-    cluster_info.set_ec2_settings(arguments.aws_region,
-                                  arguments.aws_availability_zone,
-                                  arguments.vpc_id,
-                                  arguments.subnet_id)
-
-    custom_config = None
-    if arguments.custom_config_file is not None:
-        try:
-            custom_config = open(arguments.custom_config_file).read()
-        except IOError as e:
-            sys.stderr.write("Unable to read custom config file: %s\n" %
-                             str(e))
-            usage()
-
-    cluster_info.set_hadoop_settings(arguments.master_instance_type,
-                                     arguments.slave_instance_type,
-                                     arguments.initial_nodes,
-                                     arguments.max_nodes,
-                                     custom_config,
-                                     arguments.slave_request_type,
-                                     arguments.use_hbase,
-                                     arguments.custom_ec2_tags,
-                                     arguments.use_hadoop2,
-                                     arguments.use_spark)
-
-    cluster_info.set_spot_instance_settings(
-          arguments.maximum_bid_price_percentage,
-          arguments.timeout_for_request,
-          arguments.maximum_spot_instance_percentage)
-
-    cluster_info.set_stable_spot_instance_settings(
-          arguments.stable_maximum_bid_price_percentage,
-          arguments.stable_timeout_for_request,
-          arguments.stable_allow_fallback)
-
-    fairscheduler_config_xml = None
-    if arguments.fairscheduler_config_xml_file is not None:
-        try:
-            fairscheduler_config_xml = open(arguments.fairscheduler_config_xml_file).read()
-        except IOError as e:
-            sys.stderr.write("Unable to read config xml file: %s\n" %
-                             str(e))
-            usage()
-    cluster_info.set_fairscheduler_settings(fairscheduler_config_xml,
+def _create_cluster_info(arguments, api_version):
+    custom_config = _read_file(arguments.custom_config_file, "custom config file")
+    presto_custom_config = _read_file(arguments.presto_custom_config_file, "presto custom config file")
+    fairscheduler_config_xml = _read_file(arguments.fairscheduler_config_xml_file, "config xml file")
+    ssh_public_key = _read_file(arguments.customer_ssh_key_file, "customer ssh key file")
+
+    cluster_info = None
+    if api_version >= 1.3:
+        cluster_info = ClusterInfoV13(arguments.label, api_version)
+        cluster_info.set_cluster_info(aws_access_key_id=arguments.aws_access_key_id,
+                         aws_secret_access_key=arguments.aws_secret_access_key,
```

Comment:
This is indented wrong.

Indent with twice the tab size or line up with the argument in the last line. https://www.python.org/dev/peps/pep-0008/#indentation